DuyH2 Verify Car models for sale which correspond with Truck Bed Length in body style tab should be displayed when the user click on Long checkbox

DuyH2 opened the pull request:
- Delete SauceLab folder and component
- Delete old demo.java
+ Add CarForSalePage
+ Add CarForSaleTruckPage
+ Add CarForSaleTruckDetailPage
* Modifed CarForSaleSearchLocationComponent
* Modified CustomKeyword
* Fix name of KbbBasePage
* Modified FirefoxBrowserSetup and ChromeDriverSetup
* Modified configs.properties
* Modified pom.xml

Commit:
- Delete SauceLab folder and component
- Delete old demo.java
+ Add CarForSalePage
+ Add CarForSaleTruckPage
+ Add CarForSaleTruckDetailPage
* Modifed CarForSaleSearchLocationComponent
* Modified CustomKeyword
* Fix name of KbbBasePage
* Modified FirefoxBrowserSetup and ChromeDriverSetup
* Modified configs.properties
* Modified pom.xml

namnguyen-itvn commented:
@hailong2000 pls review this PR

namnguyen-itvn commented:
@Datvh1422 pls review this PR

Datvh1422 commented:
Review this testcase:
Coding convention: Fine
Verify Car models for sale which correspond with Truck Bed Length in body style tab should be displayed when the user click on Long checkbox.
Test run Ok
Stable, Testcase name ok, Have Function Assert to verify TestCase.

hailong2000 commented:
- Coding convention: fine
- test run OK
- Have not used Data Driven

Commit:
* Update access modified for WebElement
* Remove unused WebElement

Commit:
- remove phpAccount since there is no reason to use it
- remove Report folder since there is no reason to use it
- remove phpaccountdata since there is no reason to use it

hailong2000 commented on demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java:
```diff
@@ -146,6 +147,101 @@ public CustomKeyword click(WebElement element){
+
+    public void waitForPageToLoad() throws InterruptedException
+    {
+        Thread.sleep(4000);
```
shouldn't use Thread.sleep in there

Commit:
no message

Commit:
Merge branch 'develop' into DuyH2-Verify-Car-models-for-sale-which-correspond-with-Truck-Bed-Length-in-body-style-tab-should-be-displayed-when-the-user-click-on-Long-checkbox

# Conflicts:
#	demo/src/test/java/com/example/test/core/configs.properties
#	demo/src/test/java/com/example/test/core/datadriven/ShopByPrice.java
#	demo/src/test/java/com/example/test/core/keywork/CustomKeyword.java
#	demo/src/test/java/com/example/test/pages/KBBMOOCProject/KbbBasePage.java
#	demo/src/test/java/com/example/test/tests/KBBMOOCProject/CarForSaleTests/CarForSaleSearchLocationComponent.java

Commit:
* update CustomKeyword
* update CarForSaleTruckPage
* update KbbBaseTest
* update KbbBasePage
